docs/src/main/tut/datatypes/ior.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: docs
+title:  "Ior"
+section: "data"
+source: "core/src/main/scala/cats/data/Ior.scala"
+scaladoc: "#cats.data.Ior"
+---
+# Ior
+
+`Ior` represents an inclusive-or relationship between two data types.
+This makes it very similar to the [`Either`](either.html) data type, which represents an `Xor` relationship.
```

Comment:
I'm a bit hesitant about this `Xor` reference. I think that this might be confusing, as Cats no longer has an `Xor` type. For consistency with the previous sentence, it might be better to write this as "which represents an exclusive-or" relationship.

In logic/circuits, this is often written as `XOR`, which we could potentially use, but I think that exclusive-or is more straightforward.